src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -33,6 +34,8 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
     // A list of contigs in this graph
     val contigs: List<String>
 
+    val checksum: String by lazy { checksum() }
```

Comment:
Just to clarify:  The checksum value is stored in the graph the first time someone queries for it?